tasks/README.md

```diff
@@ -69,6 +69,8 @@ The preferred style of the keys should be `snake_case`.
 
 **extensions**: A hash of extensions to the task spec used by a Specific Task Runner. Each key at the top level should be the name of the Task Runner the extension is used by. Task Runners should not read extensions outside of their own namespace.(rev 3)
 
+**remote**: Default `false`. All implementation of this task operate on a remote target using the `target` metaparam.
```

Comment:
does this require a `rev 4` annotation? (same question also on for the same change below)

Reply:
yes, I stopped adding `rev 4` so I wouldn't have to change if it something else took 4.